server/app/admin.py

```diff
@@ -1,32 +1,85 @@
 from django.contrib import admin
-from .models import Users, Profiles, Tasks, Bids, Payments
+from .models import Tasks, Bids, Payments
+from django.contrib.auth.admin import UserAdmin
+
+from .models import Users
 
 
-# Users Table Interface
 @admin.register(Users)
-class UsersAdmin(admin.ModelAdmin):
-    list_display = ("user_id", "email", "mobile", "created_at",
-                    "updated_at", "last_login", "status", "user_role")
-    list_filter = ("user_role", "status")
+class UserAdminConfig(UserAdmin):
+    model = Users
+    list_display = (
+        "username",
+        "email",
+        "first_name",
+        "last_name",
+    )
+    list_filter = ("is_bidder", "is_poster")
     search_fields = ("email", "mobile")
     date_hierarchy = "created_at"
     ordering = ("-created_at",)
+    fieldsets = (
+        (None, {"fields": ("username", "password")}),
+        ("Personal info", {"fields": ("first_name", "last_name", "email", "mobile")}),
+        (
+            "Permissions",
+            {
+                "fields": (
+                    "is_active",
+                    "is_staff",
+                    "is_superuser",
+                    "groups",
+                    "user_permissions",
+                )
+            },
+        ),
+        ("Important dates", {"fields": ("last_login", "date_joined")}),
+    )
 
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
+                "fields": (
+                    "username",
+                    "email",
+                    "password1",
+                    "password2",
+                    "first_name",
+                    "last_name",
+                    "is_active",
+                    "is_staff",
+                    "is_bidder",
+                    "is_superuser",
+                    "groups",
+                    "user_permissions",
+                ),
+            },
+        ),
+    )
 
-# Profiles Table Interface
-@admin.register(Profiles)
-class ProfilesAdmin(admin.ModelAdmin):
-    list_display = ("profile_id", "user_id", "full_name", "avatar_url", "bio")
-    search_fields = ("full_name", )
 
+# admin.site.register(Users, UserAdminConfig)
 
 # Tasks Table Interface
+
+
 @admin.register(Tasks)
 class TasksAdmin(admin.ModelAdmin):
-    list_display = ("task_id", "owner_id", "title", "category",
-                    "description", "location", "budget",
-                    "estimated_time", "deadline", "status", "created_at",
-                    "updated_at")
+    list_display = (
+        "task_id",
+        "poster_id",
+        "title",
+        "category",
+        "description",
+        "location",
```

Comment:
ideally, the location should be json format, which contains Suburb and State field. 

{
"Suburb": xxx,
"State": xxx, 
// extend to other fields in the future
}